test/framework/src/main/java/org/elasticsearch/index/mapper/MapperServiceTestCase.java

```diff
@@ -101,7 +100,7 @@
  * mapping. Useful when you don't need to spin up an entire index but do
  * need most of the trapping of the mapping.
  */
-public abstract class MapperServiceTestCase extends ESTestCase {
+public abstract class MapperServiceTestCase extends FieldTypeTestCase {
```

Comment:
Perhaps this is no longer required given my comment above. We can still make this change as a follow-up if needed.

Reply:
I would keep it since it is needed to standardize tests against `AbstractScriptFieldType` and `RankFeatureMetaFieldMapperTests` and `DataTierFieldTypeTests`. Those where not extending `FieldTypeTestCase` but `MapperServiceTestCase`.

Reply:
sounds good.